### PR TITLE
Update people-example.md

### PR DESCRIPTION
**Pull request by polio77**
adding Working With description

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[77d45dd](https://github.com/polio77/microsoft-graph-docs/pull/4268/commits/77d45ddd18fb2b3936d10329cffec5894ccf81ad)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/people-example.md](https://github.com/polio77/microsoft-graph-docs/blob/polio77-patch-1/concepts/people-example.md) | :warning:Warning | | [Details](#user-content-concepts%2Fpeople-example.md)

<a name="concepts%2Fpeople-example.md"></a>
### [concepts/people-example.md](https://github.com/polio77/microsoft-graph-docs/blob/polio77-patch-1/concepts/people-example.md)
 - **Line 948**: **[Warning]** Invalid file link:(~/images/working-with.png).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C28%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903281146278470-4268%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=c6rmBm0X%2FUIp2uTAJ468ykOJwsHYRit%2B4HSpo8OtFXs%3D&st=2019-03-28T11%3A45%3A08Z&se=2019-04-28T11%3A50%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by polio77** [comment]
@Lauragra Hi Laura - what action is required on my part to fix this? You added the 'do not merge' label, but I can't figure out why

**Comment by Lauragra** [comment]
Hi @polio77 , did you intend to close this PR? Adding the do not merge label is a standard procedure so that we don't merge in content that hasn't had a review. I was planning to review this PR and give any feedback before we publish. Do you still want to publish this update? Thanks.

**Comment by polio77** [comment]
Hi Laura - sorry i didn't mean to close this (I read 'close' as close the comment dialog...), I still want it to be published. Thanks for explaining the process. Reopening.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[77d45dd](https://github.com/polio77/microsoft-graph-docs/pull/4268/commits/77d45ddd18fb2b3936d10329cffec5894ccf81ad)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/people-example.md](https://github.com/polio77/microsoft-graph-docs/blob/polio77-patch-1/concepts/people-example.md) | :warning:Warning | | [Details](#user-content-concepts%2Fpeople-example.md)

<a name="concepts%2Fpeople-example.md"></a>
### [concepts/people-example.md](https://github.com/polio77/microsoft-graph-docs/blob/polio77-patch-1/concepts/people-example.md)
 - **Line 948**: **[Warning]** Invalid file link:(~/images/working-with.png).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C29%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903291635421658-4268%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=q66xTjpULAqPNBIVsz0fIFAG5oW4qXm1eMhFzc5NH4s%3D&st=2019-03-29T16%3A34%3A28Z&se=2019-04-29T16%3A39%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by polio77** [comment]
Hi @Lauragra - is there anything I can do to expedite publishing of this update? There's a time aspect to this as there was a recent escalation & publishing this update is a response to that

**Comment by Lauragra** [comment]
@polio77 , are you a member of the docs contrbutors group, and have you joined your GitHub account with your alias?

**Comment by polio77** [comment]
@Lauragra Indeed both of these are the case - I'm on the contributors group & my alias (paulmck) is mapped to my GitHub account

**Comment by polio77** [comment]
@Lauragra Hi Laura - any update? As I mentioned above, this needs to be published to address an escalation, and we're already past the deadline

**Comment by Lauragra** [comment]
I'll publish it today. I'll assume that this PR doesn't need a tech review. Thx.

**Comment by polio77** [comment]
Perfect, thanks Laura - you're correct, there isn't a tech review required for this

File: concepts/people-example.md [hunk 1]
```diff
@@ -942,3 +942,32 @@ GET https://graph.microsoft.com/v1.0/me/people?$search="tylerle@example.com"  //
 GET https://graph.microsoft.com/v1.0/me/people?$search="tiler"                //fuzzy match with Tyler's name
 GET https://graph.microsoft.com/v1.0/me/people?$search="tyler lee"            //matches Tyler's name. Note the quotes to enclose the space.
 ```
+
+### Delve “Working With” feature implementation
+ 
+![Image of working with relationships](images/working-with.png "Image of Working With relationships")
```
Comment (Lauragra) [inline concepts/people-example.md]: @polio77 , do we have the image file in the repo? I'm not able to preview the image. Thanks.

Comment (polio77) [reply concepts/people-example.md]: I think this is it: https://github.com/polio77/microsoft-graph-docs/commit/604122193344973827da405858f89fbea46f6e47

File: concepts/people-example.md [hunk 2]
```diff
@@ -942,3 +942,32 @@ GET https://graph.microsoft.com/v1.0/me/people?$search="tylerle@example.com"  //
 GET https://graph.microsoft.com/v1.0/me/people?$search="tiler"                //fuzzy match with Tyler's name
 GET https://graph.microsoft.com/v1.0/me/people?$search="tyler lee"            //matches Tyler's name. Note the quotes to enclose the space.
 ```
+
+### Delve Working With feature implementation
+ 
+There must be a public relationship between the profile owner and the other people in order for those people to show up on the profile owner's list. The following illustration shows a User A, an index of relationships with other users (User B), and a public profile showing a subset of user relationships.
+
```
Comment (Lauragra) [inline concepts/people-example.md]: We need to add a more specific description of the image for non-sighted users. See if this works or if it needs some tweaks?

File: concepts/people-example.md [hunk 3]
```diff
@@ -942,3 +942,32 @@ GET https://graph.microsoft.com/v1.0/me/people?$search="tylerle@example.com"  //
...
+- Sending emails to each other where the name of the other person is in the TO line
+- Inviting users to meetings by including their name in the calendar invite 
+ 
+Examples of public interaction include:
+- Sending or receiving emails to/from each other as part of a public group 
+- Inviting users to meetings as part of group, or where more than X people are invited
+ 
+The ranking doesn’t change based on who User A is (the person looking at someone else's page). The ranking is determined by the interaction level between User B (profile owner) and User C (person showing up on profile owner's list).
+ 
+In order for User C to appear, the profile owner must be in a relatively small group/DL with that user that is public (meaning the membership list is available in the directory).
```
Comment (Lauragra) [inline concepts/people-example.md]: Is User C a reference to something in the image? It would be good to clarify what we're referring to here?

**Comment by Lauragra** [comment]
@polio77 , pls. take a look at my last remaining comments when you get a chance? Thanks.

File: concepts/people-example.md (hunk 2 above)
Comment (polio77) [reply concepts/people-example.md]: I think your description is perfect, thanks Laura

File: concepts/people-example.md (hunk 3 above)
Comment (polio77) [reply concepts/people-example.md]: 'User C' is someone who doesn't meet the criteria to appear in the list of 'User A' - they are purposely excluded from the image to reinforce that point that external parties cannot inadvertantly make their way onto A's list

Comment (polio77) [reply concepts/people-example.md]: Hi Laura, I need to make one change - removing the word 'Delve' from the section title. Please revert it back to me if necessary (or remove that word)

**Comment by polio77** [comment]
Awesome, thanks so much @Lauragra

**Pull request merged**